### PR TITLE
[SOAR-10513] Palo Alto Cortex XDR - Fixed descriptions, examples

### PR DESCRIPTION
**Pull request by igorski-r7**
### Description

Describe the proposed changes:

  - Changed description for `Get XQL Query Results` action, `Get Query Results` trigger
  - Added example output field to `Get Query Results` trigger
  - Changed the query examples for `Get XQL Query Results`, and `Get Query Results` in order not to brake the `help.md` table

## PR Requirements

Developers, verify you have completed the following items by checking them off:

#### Unit Tests

Review our documentation on [generating](https://docs.rapid7.com/insightconnect/unit-test-generation) and [writing](https://docs.rapid7.com/insightconnect/unit-test-primer) plugin unit tests

- [ ] Unit tests written for any new or updated code

#### In-Product Tests

If you are an InsightConnect customer or have access to an InsightConnect instance, the following in-product tests should be done:

- [ ] Screenshot of job output with the plugin changes
- [ ] Screenshot of the changed connection, actions, or triggers input within the InsightConnect workflow builder

### Style

Review the [style guide](https://docs.rapid7.com/insightconnect/style-guide/)

- [ ] For dependencies, pin [OS package](https://docs.rapid7.com/insightconnect/style-guide/#dockerfile) and [Python package](https://docs.rapid7.com/insightconnect/style-guide/#requirements.txt) versions
- [ ] For security, set least privileged account with ``USER nobody`` in the ``Dockerfile`` when possible
- [ ] For size, use the [slim SDK images](https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide) when possible: ``komand/python-3-37-slim-plugin`` and ``komand/python-3-37-plugin``
- [ ] For error handling, use of [PluginException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations/#plugin-exceptions) and [ConnectionTestException](https://docs.rapid7.com/insightconnect/error-handling-in-integrations#connection-exceptions)
- [ ] For logging, use [self.logger](https://docs.rapid7.com/insightconnect/sdk-guide/#logging)
- [ ] For docs, use [changelog style](https://docs.rapid7.com/insightconnect/style-guide/#changelog)
- [ ] For docs, validate markdown with ``make validate`` which calls ``mdl`` to lint ``help.md``

### Functional Checklist
- [ ] Work fully completed
- [ ] Functional
  - [ ] Any new actions/triggers include JSON [test files](https://docs.rapid7.com/insightconnect/style-guide/#tests) in the `tests/` directory created with `icon-plugin run -c sample $action > tests/$action.json`
  - [ ] Tests should all pass unless it's a negative test. Negative tests have a naming convention of `tests/$action_bad.json`
  - [ ] Unsuccessful tests should fail by raising an exception causing the plugin to die and an object should be returned on successful test
  - [ ] Add functioning test results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -T tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -T all --debug --jq` (use PR format at end)
  - [ ] Add functioning run results to PR, sanitize any output if necessary
    * Single action/trigger `icon-plugin run -R tests/example.json --debug --jq`
    * All actions/triggers shortcut `icon-plugin run -R all --debug --jq` (use PR format at end)

### Assessment

You must validate your work to reviewers:

1. Run `make validate` and make sure everything passes
2. Run the assessment tool: `icon-plugin run -A -R all -T all`. For single action validation: `icon-plugin run -A -R tests/my_action.json -T tests/my_action.json`
3. Copy (`icon-plugin ... | pbcopy`) and paste the output in **a new post** on this PR
4. Add required screenshots from the In-Product Tests section

### CODE REVIEW

File: plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/actions/get_xql_query_results/schema.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ class GetXqlQueryResultsInput(insightconnect_plugin_runtime.Input):
     "tenants": {
       "type": "array",
       "title": "Tenants",
-      "description": "String that represents additional information regarding the action",
+      "description": "List of string used when querying tenants managed by Managed Security Services Providers (MSSP)",
```
Comment (jkaluza-rapid7) [inline plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/actions/get_xql_query_results/schema.py:56]:
```suggestion
      "description": "List of strings used when querying tenants managed by Managed Security Services Providers (MSSP)",
```

File: plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/triggers/get_query_results/schema.py [hunk 2]
```diff
@@ -49,7 +49,7 @@ class GetQueryResultsInput(insightconnect_plugin_runtime.Input):
     "tenants": {
       "type": "array",
       "title": "Tenants",
-      "description": "String that represents additional information regarding the action",
+      "description": "List of string used when querying tenants managed by Managed Security Services Providers (MSSP)",
```
Comment (jkaluza-rapid7) [inline plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/triggers/get_query_results/schema.py:52]:
```suggestion
      "description": "List of strings used when querying tenants managed by Managed Security Services Providers (MSSP)",
```

File: plugins/palo_alto_cortex_xdr/plugin.spec.yaml [hunk 3]
```diff
@@ -1144,10 +1144,10 @@ actions:
         description: String of the XQL query
         type: string
         required: true
-        example: dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3
+        example: dataset=xdr_data I fields event_id, event_type, event_sub_type I limit 3
       tenants:
         title: Tenants
-        description: String that represents additional information regarding the action
+        description: List of string used when querying tenants managed by Managed Security Services Providers (MSSP)
```
Comment (jkaluza-rapid7) [inline plugins/palo_alto_cortex_xdr/plugin.spec.yaml:1150]:
```suggestion
        description: List of strings used when querying tenants managed by Managed Security Services Providers (MSSP)
```

File: plugins/palo_alto_cortex_xdr/plugin.spec.yaml [hunk 4]
```diff
@@ -1225,13 +1225,13 @@ triggers:
...
```
Comment (jkaluza-rapid7) [inline plugins/palo_alto_cortex_xdr/plugin.spec.yaml:1231]:
```suggestion
        description: List of strings used when querying tenants managed by Managed Security Services Providers (MSSP)
```

**Review by jkaluza-rapid7 — CHANGES_REQUESTED** [review]

**Review by jkaluza-rapid7 — APPROVED** [review]

File: plugins/palo_alto_cortex_xdr/plugin.spec.yaml [hunk 5]
```diff
@@ -1144,10 +1144,10 @@ actions:
         description: String of the XQL query
         type: string
         required: true
-        example: dataset=xdr_data | fields event_id, event_type, event_sub_type | limit 3
+        example: dataset=xdr_data I fields event_id, event_type, event_sub_type I limit 3
```
Comment (ablakley-r7) [inline plugins/palo_alto_cortex_xdr/plugin.spec.yaml:1147]: Does the letter I work as well as the | character for queries?

Comment (igorski-r7) [reply plugins/palo_alto_cortex_xdr/plugin.spec.yaml:1147]: Nope. Actually, it just seems to be looking the same way as `|` character. Putting that in quotes did change nothing. It was breaking the markdown table as well 😞

**Review by ablakley-r7 — APPROVED** [review]

**Pull request merged**